app/routines/search_users.rb

```diff
@@ -83,9 +83,11 @@ def exec(query, options={})
       end
 
       with.keyword :uuid do |uuids|
-        sanitized_uuids = sanitize_strings(uuids, append_wildcard: options[:admin],
-                                            prepend_wildcard: options[:admin])
-        users = users.where{uuid.like_any sanitized_uuids}
+        if not uuids[0].include? '-' and uuids[0].to_s.length == 8
+            users = users.where(:uuid => uuids[0] + '-0000-0000-0000-000000000000'..uuids[0] + '-ffff-ffff-ffff-ffffffffffff')
```

Comment:
Can you get rid of the hash rocket `=>`? I've personally been advocating for using the new hash syntax when possible: (`uuid: partial_uuid + ...`)